src/lib.rs

```diff
@@ -381,6 +424,23 @@ impl<T> TelegrafUnwrap<T> for Option<T> {
 mod tests {
     use super::*;
 
+    #[test]
+    fn can_create_point_ts() {
```

Comment:
thanks for this test. Can you also add a test for one that has a timestamp but no tags?